docs/reference/commandline/update.md

```diff
@@ -48,11 +48,11 @@ options on a running or a stopped container. On kernel version older than
 4.6, you can only update `--kernel-memory` on a stopped container or on
 a running container with kernel memory initialized.
 
-## Examples
+# Examples
```

Comment:
This is not correct; a page should only have a single H1, not multiple (same for the headings below)